menu/src/MenuWrapper.tsx

```diff
@@ -39,9 +43,16 @@ const MenuWrapper: React.FC = () => {
   //Change page back to Main when closed
   useEffect(() => {
     if (visible) return;
+
     const changeTimer = setTimeout(() => {
       setPage(txAdminMenuPage.Main);
     }, 750);
+
+    if (playersFilterIsTemp) {
+      setPlayerFilter("");
+      setPlayersFilterIsTemp(false);
+    }
+
     return () => clearInterval(changeTimer);
   }, [visible]);
```

Comment:
Missing dependent state in dep array. The setter function is stable and can be ignored from deps.

```suggestion
  }, [visible, playersFilterIsTemp]);
```